src/js/other-websites/inpage_menu.js

```diff
@@ -146,7 +146,7 @@ async function fillTargetWithRelayAddress(generateClickEvt) {
 
   if (serverStoragePref) {
     // Update server info with site usage
-    await browser.runtime.sendMessage({
+    browser.runtime.sendMessage({
```

Comment:
Performance fix: dropped the `await` here as it shouldn't block the user from injecting the mask into the field.

Reply:
Nice!